ahmed-yara-sara/src/App.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import "./App.css";
+import Navbar1 from "./components/Navbar";
+import Main from "./components/Main";
+import FooterBar from "./components/Footer";
+import { useState } from "react";
```

Comment:
you can make it like this
```suggestion
import React, { useState } from "react";
import "./App.css";
import Navbar1 from "./components/Navbar";
import Main from "./components/Main";
import FooterBar from "./components/Footer";
```